DigitalLearningSolutions.Web/ViewModels/Supervisor/SupervisorDelegateDetailViewModel.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.Supervisor
+{
+    using DigitalLearningSolutions.Data.Models;
+    using DigitalLearningSolutions.Data.Models.Supervisor;
+
+    public class SupervisorDelegateDetailViewModel : BaseSearchableItem

```

Comment:
This is a bit of a weird case because the existing view model was a model from the data project. I just needed to add a Page parameter to it, and we decided in the peer review this was the best way. My initial solution inherited SupervisorDelegateDetail instead, but required some weird constructors throughout the class hierarchy.